bot/interact.go

```diff
@@ -207,3 +211,20 @@ func generateReportMessageDialog() slack.Dialog {
 		Elements:    elements,
 	}
 }
+
+func buildPublisherInput() *slack.DialogInputSelect {
+	publisherOptions := []slack.DialogSelectOption{
+		{
+			Label: "End Company",
+			Value: "End Company",
+		},
+		{
+			Label: "Recruitment Agency",
+			Value: "Recruitment Agency",
```

Comment:
I am a bit concerned on the final length. I would like to suggest we short `End Company` and `Recruitment Agency` values so they fit OK on the published offer. 

Perhaps I will only add it in case it is a Recruitment Agency, and short form such as `R. Agency` or similar.

WDYT?

Reply:
Agree. Let me think a bit and come up with shorter values. 

On the conditional display, I think that from a user perspective and for searching purposes, it's more comfortable to have both fields, that way a user will be able to search the ones that are **only** from an end company. by typing in the search box "End company" otherwise if a user wants to see only offers published by end companies, the search would be -"end company" which, in my opinion is less intuitive.

What do you think?

![image](https://user-images.githubusercontent.com/13377065/124083526-42536d00-da4e-11eb-9ec7-b06f4010e19d.png)

Reply:
What about `Agency` and `Employer`?

Reply:
Fair enough! I think we can move forward with your suggestion 👍 🚀